site2/website/scripts/replace.js

```diff
@@ -32,6 +32,8 @@ function binaryReleaseUrl(version) {
 function connectorReleaseUrl(version) {
     if (version.includes('incubating')) {
         return `https://archive.apache.org/dist/incubator/pulsar/pulsar-${version}/apache-pulsar-io-connectors-${version}-bin.tar.gz`
+    } else if (version >= '2.3.0') {
```

Comment:
have you built the website locally to make sure the change work? 

frankly speaking, I am not sure `version >= '2.3.0'` work

Reply:
@liketic. FYI. file `site2/README.md` contains how to run it locally.

Reply:
Thanks @sijie @jiazhai , I can run the website through:
```
cd website
yarn install
yarn start
```
However, the variables in the documents are not replaced. And I also attempted to run ```docker-build-site.sh``` to rebuild the website and never get a successful build.  

I can debug the method ```connectorReleaseUrl``` with my local ```node``` environment and seems like it works as I expected. May I ask your help for how to build the website correctly? 

Thank you in advance.

Reply:
You can checkout the build script : https://github.com/apache/pulsar/blob/master/site2/tools/docker-build-site.sh

You can build the content and run the replace js to replace the variables and verify the content is displaying the download links correctly.